### ISSUE TITLE
Generated Hash was not used in password creation.

### ISSUE THREAD

**Issue by jagadeeshtummala** [KalleHallden/pwManager#19]
hi @KalleHallden ,

recently found this repo for password manager. correct me if i am wrong, the generated Hash Value (raw_hex variable) in the hash_maker.py file is not used in the generation of random password. so, what's the use of calculating Hash values using appname and plaintext given by user. please mention if the hash values are used else where. 
thank you,

**Comment by HacDan** [comment]
@jagadeeshtummala You're correct. `raw_hex` is only used by the `num` variable and the `num` variable isn't used at all. 

This is mentioned in Issue #2 as well.
